service/auth_test.go

```diff
@@ -110,3 +115,86 @@ func Test_authClient_ValidateToken(t *testing.T) {
 		})
 	}
 }
+
+func Test_authClient_ValidateBlacklistToken(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockUserRepo := mockRepo.NewMockUser(ctrl)
+	authSvc := NewAuthClient("secret", nil, mockUserRepo)
+
+	user := entity.User{
+		Id: uuid.New(),
+	}
+	params := entity.GetByTokensRequest{}
+
+	type args struct {
+		ctx    context.Context
+		params *entity.GetByTokensRequest
+	}
+	tests := []struct {
+		name     string
+		mocks    []*gomock.Call
+		args     args
+		wantErr  bool
+		wantResp bool
+	}{
+		{
```

Comment:
I think this test case is obsolete now, since the return is just error or success, no both nil for user and error